document/docs/en/config/options/index.mdx

```diff
@@ -0,0 +1,136 @@
+# Options Configuration
```

Comment:
Options and Configuration have similar meanings, we usually do not use “Options Configuration”, just `Options` is enough.

```suggestion
# Options
```